src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -46,4 +46,8 @@
     <Copy SourceFiles="@(_Packages)" DestinationFolder="../local-packages" />
     <Message Text="Copied sql .nupkg to local-packages" Importance="high" />
   </Target>
+  <Target Name="RemoveLocalPackages" BeforeTargets="Build" Condition="'$(Configuration)' == 'Debug'">
```

Comment:
Why are you removing this folder anyways? The folder isn't the problem - the package there will get overwritten during the build. It's the nuget cache (%USERPROFILE%/.nuget for Windows) that's the thing that needs to be cleared out.

Reply:
$(NugetPackageRoot)/$(PackageId.ToLower())/99.99.99 should do?